docs/index.rst

```diff
@@ -18,6 +18,40 @@ Install from pypi::
 
     python -m pip install django-pg-migration-tools
 
+Background
+==========
+
+In the past, traditional applications would do schema changes during scheduled
+"maintenance windows." During this time, the application would stop all
+traffic, and developers could make changes to the database or servers without
+worrying about users.
+
+In the context of modern systems requiring 24/7 availability, this is no longer
+possible.
+
+Django's migration system usually picks the easiest way to make a schema
+change, but that isn't always safe. In fact, some operations Django does can
```

Comment:
It would be good to define "safe" here, particularly since many operations are named "…Safer…". The rest of the "Background" section explains what the implications and requirements of being safe are, but it would be good to have a short summary first. E.g.

> By "safe", we mean that the migration operation can run during normal application operating hours without interrupting the availability of the database or causing errors while using the database.

Reply:
Good idea. I have added an info component here: 6d6763313db93e0452903c00a7b8ff0a9da78893